WebRoot/js/ajax/dwt/widgets/DwtMenu.js

```diff
@@ -1156,7 +1156,7 @@ function(x, y, kbGenerated) {
 		DwtMenu._activeMenuUp = true;
 	}
 
-	this.parent && this.parent.addClassName && this.parent.addClassName(DwtControl.SUB_MENU_ACTIVE);
+	this.parent && this.parent instanceof DwtButton && this.parent.addClassName && this.parent.addClassName(DwtControl.SUB_MENU_ACTIVE);
```

Comment:
Dwt.Combobox and Dwt.Select are using Dwt.Menu as well, have we checked this is not breaking that